.python_package/noxfile.py

```diff
@@ -0,0 +1,123 @@
+# GridTools
+#
+# Copyright (c) 2014-2021, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import configparser
+import pathlib
+import shutil
+
+import nox
+
+nox.options.sessions = ["test_src", "test_wheel"]
+
+
+def prepare(session: nox.Session):
+    session.install("cmake>=2.18.1")
+    session.install("ninja")
+    build_path = session.cache_dir.joinpath("build").absolute()
+    build_path.mkdir(exist_ok=True)
+    install_path = pathlib.Path(".").absolute() / "src" / "gridtools" / "data"
+    source_path = pathlib.Path("..").absolute()
+    with session.chdir(build_path):
+        session.run(
+            "cmake",
+            "-DBUILD_TESTING=OFF",
+            "-DGT_INSTALL_EXAMPLES:BOOL=OFF",
+            f"-DCMAKE_INSTALL_PREFIX={install_path}",
+            "-GNinja",
+            str(source_path),
+        )
+        session.run("cmake", "--install", ".")
+        session.log("installed gridttols sources")
+    version_path = source_path / "version.txt"
+    setup_path = pathlib.Path(".") / "setup.cfg"
+    config = configparser.ConfigParser()
```

Comment:
Probably you're already aware, but `configparser` removes all comments and formatting from the original file. It's not very important in this case, though, but it is not nice. The only alternative I know which  preserves format is ConfigUpdate (https://pypi.org/project/ConfigUpdater/) but is a third-party project